radio/src/opentx.h

```diff
@@ -340,9 +340,7 @@ inline bool SPLASH_NEEDED()
 #endif
 }
 
-#if defined(PCBHORUS)
-  #define SPLASH_TIMEOUT               0 /* we use the splash duration to load stuff from the SD */
-#elif defined(PCBTARANIS)
+#if defined(PCBHORUS) || defined(PCBTARANIS) || defined(PCBNV14)
```

Comment:
This was the only other oddity... normally when PCBHORUS and PCBNV14 are together, they can be replaced with COLORLCD, making it more generic, and PCBTARANIS is every single B&W radio ... so what is this really conditional on now, when would ELSE ever trigger? Perhaps it only needs to be conditional on SPLASH?

Reply:
Historically all arm 9x and earlier mega radio

Reply:
Thanks, ok, so the else can indeed go as we've dropped those radios, and I guess `#if defined(SPLASH)` will do then.